src/pages/login.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState, useEffect } from "react";
+import { LoginButton } from "@inrupt/solid-ui-react";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import InputLabel from "@mui/material/InputLabel";
+import MenuItem from "@mui/material/MenuItem";
+import FormControl from "@mui/material/FormControl";
+import Select from "@mui/material/Select";
+import Container from "@mui/material/Container";
+import Stack from "@mui/material/Stack";
+import TextField from "@mui/material/TextField";
+import Box from "@mui/material/Box";
+
+const providers = [
+  { title: "Inrupt Pod Spaces", url: "https://broker.pod.inrupt.com/" },
+  { title: "inrupt.net", url: "https://inrupt.net/" },
+  { title: "solidcommunity.net", url: "https://solidcommunity.net/" },
+  { title: "Solid Web", url: "https://solidweb.org/" },
+  { title: "Trinpod", url: "https://trinpod.us/" },
+  { title: "Oxford", url: "https://test.pod.ewada.ox.ac.uk" },
+];
+
+export default function Login() {
+  const [currentUrl, setCurrentUrl] = useState("https://localhost:3000");
+  const [provider, setProvider] = useState("");
+
+  useEffect(() => {
+    setCurrentUrl(window.location.origin);
+  }, [setCurrentUrl]);
+
+  return (
+    <div>
+      <Container maxWidth="sm">
+        <Typography component="h2" variant="h5" color="inherit" noWrap>
+          Sign in
+        </Typography>
+        <Typography
+          component="h2"
+          variant="body1"
+          color="inherit"
+          noWrap
+          sx={{ flex: 1 }}
+        >
+          Select your pod provider or input custom server
```

Comment:
Users should be able to also only provide their WebID. Their pod provider can be determined from that. If there are multiple then a dropbox should be shown picking the desired one.

Reply:
It's best to offer both solution. Mostly because we have no proof at the moment that users prefer one or the other. And even then, the majority (however defined) should not be the only ones considered here.